test/erlfmt_SUITE.erl

```diff
@@ -1303,10 +1307,42 @@ contains_pragma_string(String) ->
     end.
 
 insert_pragma_string(String) ->
-    {ok, StringWithPragma, []} = erlfmt:format_string(String, [{pragma, insert}]),
+    {ok, #{formatted := StringWithPragma, warnings := []}} =
+        erlfmt:format_string(String, [{pragma, insert}]),
     %% check that insert_pragma_nodes doesn't insert a pragma, when one has already been inserted.
-    ?assertEqual(
-        {ok, StringWithPragma, []},
+    ?assertMatch(
+        {ok, #{formatted := StringWithPragma, warnings := []}},
         erlfmt:format_string(StringWithPragma, [{pragma, insert}])
     ),
     StringWithPragma.
+
+overlong_warning(Config) when is_list(Config) ->
```

Comment:
Really love how this is tested, thank you so much for including this.

Reply:
Maybe it would be good to add a unicode example, to test that we use the correct way of checking length.

Reply:
I guess the point would be to check that we don't get the warning, because we are counting glyphs and not bytes. That sounds like a good idea. Do you have a good example to go with?
I was thinking of using a pangram, either "色は匂へど 散りぬるを 我が世誰ぞ 常ならむ 有為の奥山 今日越えて 浅き夢見じ 酔ひもせず" or "Nne, nna, wepụ he’l’ụjọ dum n’ime ọzụzụ ụmụ, vufesi obi nye Chukwu, ṅụrịanụ, gbakọọnụ kpaa, kwee ya ka o guzoshie ike; ọ ghaghị ito, nwapụta ezi agwa." (but broken into multiple lines).
The first one is nicer to look at, but the second one is trickier.
There are examples (like this horrible thing: "A͋͠r͍̞̫̜͌ͦ̈́͐ͅt̼̭͞h́u̡̙̞̘̙̬͖͓rͬͣ̐ͮͥͨ̀͏̣") where both erlang and python3 report a length of 39, which I'm not sure can be handled.

Reply:
Anyone will do, we just need at least one normal example.
I see you have added one, it looks good to me.